poolakey/src/main/java/ir/cafebazaar/poolakey/billing/connection/ReceiverBillingConnection.kt

```diff
@@ -202,6 +216,62 @@ internal class ReceiverBillingConnection(
         }.run(::sendBroadcast)
     }
 
+    private fun getFeatureConfig(
+        callback: GetFeatureConfigCallback.() -> Unit
+    ) {
+        featureConfigCallback = callback
```

Comment:
You need version check here

Reply:
this function just sendBroadcast for getting the feature config , we check the version before it calls.